<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>application_description</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="Application_description_files/libs/clipboard/clipboard.min.js"></script>
<script src="Application_description_files/libs/quarto-html/quarto.js"></script>
<script src="Application_description_files/libs/quarto-html/popper.min.js"></script>
<script src="Application_description_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Application_description_files/libs/quarto-html/anchor.min.js"></script>
<link href="Application_description_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Application_description_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Application_description_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Application_description_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Application_description_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">



<section id="project-title" class="level1">
<h1>PROJECT TITLE</h1>
<p><strong>Metabolic Signatures in pre-diabetes: Decoding Incretin and Glucagon Pathways and Their Interaction with Metabolic Traits in cardiometabolic disease progression</strong></p>
</section>
<section id="brief-project-description-incl.-choice-of-database-maximum-2000-characters-including-spaces-line-breaks-and-special-characters" class="level1">
<h1>BRIEF PROJECT DESCRIPTION incl.&nbsp;CHOICE OF DATABASE (maximum 2,000 characters, including spaces, line breaks and special characters)</h1>
<p>This project investigates the role of natural incretin and glucagon responses as drivers of progression toward diabetes and diabetes-related complications.</p>
<p>People at high risk of type 2 diabetes are not currently being systematically captured, primarily due to heterogeneity in who progresses to diabetes. Many of the driving and compensatory protective mechanisms involved in the development of type 2 diabetes and its related complications remain to be uncovered. Evidence from cohort studies on the long-term consequences of disrupted natural incretin and glucagon responses remains limited.</p>
<p>This proposal project investigates a missing piece in the puzzle the role of natural incretin and glucagon, aiming to identify the primary drivers of progression toward diabetes and diabetes-related complications. The project will leverage the deep-phenotyped data in the Fenland study representing the UK general population and validate in the Danish ADDITION-PRO and UK biobank. The project is divided into two parts: (1) Using traditional and newer methods of causal inference, including genetic predicted and phenotypic measures, we aim to determine the causal relationship between incretin and glucagon and long-term cardiometabolic outcomes and to identify mediating pathways such as liver health, inflammation and metabolites. This will be investigated using data from the Fenland Study and validated in large population studies such as the Danish ADDITION-PRO and UK Biobank. (2) We will investigate incretin and glucagon responses in conjunction with current components of heterogeneity in the risk of metabolic dysfunction, such as liver health and inflammation, and aim to decipher the impact of each metabolic trait on long-term cardiometabolic risk.</p>
<p>By using novel epidemiological methods to characterizing peoples’ risk of progression towards cardiometabolic disease, and identifying those with a high potential benefit from earlier intensive intervention, this proposal aim to aid prevention of type 2 diabetes.</p>
<p><strong>Choice of database</strong></p>
<p>The Fenland Study (MRC Epidemiology Unit, UK) is uniquely suited as the selected database because it combines OGTT-based incretin/glucagon, genetics, proteomics, metabolomics, DEXA-based adiposity, liver function biomarkers, and long-term cardiometabolic outcomes in a general population. We will validate findings in ADDITION-PRO (Danish cohort) and UK Biobank (external validation), where applicable. Value added to the Fenland database: the project will generate and deposit (1) harmonised OGTT-derived incretin–glucagon phenotypes, (2) causal pathway maps and mediator panels (metabolites/biomarkers), (3) machine-learning metabolic clusters, and (4) prediction algorithms and code. All derived assets will be returned to Fenland within two years of generation/publication, in FAIR format, thereby enhancing database attractiveness and reusability for future cardiometabolic research.</p>
</section>
<section id="project-description-maximum-20000-characters-including-spaces-line-breaks-and-special-characters." class="level1">
<h1>PROJECT DESCRIPTION (maximum 20,000 characters, including spaces, line breaks and special characters).</h1>
<p>Successful applications describe focused projects, feasible within the budget, timeframe and the manpower requested (maximum 20,000 characters, including spaces, line breaks and special characters).</p>
<p>Please consider the following:<br>
• Describe your proposed research project in detail – including purpose, state-of-the-art, background, methods, implementation, novelty, feasibility, and the significance of the project.<br>
• In case of collaboration with another research group, its nature must be described in the project description, and the main applicant must be the leader of the project. The role of the collaborator must also be described.<br>
• Include a short paragraph of the synergy of the proposed project with ongoing project(s) and already funded activities. • You are encouraged to include and describe preliminary data.<br>
• Up to four illustrations (figures, tables, diagrams etc.) can be uploaded. Please only include illustrations relevant for the assessment of your application. Inclusion of a Gantt Chart and preliminary data as figures are welcomed. It can take up to five working days to register a new administrating institution in NORMA. The application cannot be submitted before the institution has been registered. SIDE 13/21 • In case you are submitting a project proposal, which has been submitted to NNF before, please clearly describe what has changed/improved in the application/project.<br>
• Abbreviations should be defined at the first use, and preferably a list of abbreviations should be included in the project description.</p>
<section id="specific-aim" class="level2">
<h2 class="anchored" data-anchor-id="specific-aim">Specific aim</h2>
<p>This Inter-SUSTAIN Part 2 projects aims to using The Fenland study to elucidate the role of natural incretin and glucagon responses as drivers of heterogeneity in the long-term risk of diabetes and related complications, and to map clinical and biological traits related to these responses with particular emphasis on liver health, adiposity and inflammation. Thereby, identifying patterns of metabolic dysfunction that may predict progression to type 2 diabetes and inform personalized treatment strategies.</p>
<p>The specific aims are to:</p>
<ol type="a">
<li><p>To phenotypically and genotypically investigate the causal role of natural incretin and glucagon responses to an oral glucose tolerance test (OGTT) in determining the risk of regression to normoglycemia, progression to type 2 diabetes, and the development of diabetes-related complications. This will be conducted using data from the Fenland study and the Danish ADDITION-PRO cohort, linked to comprehensive outcomes from the Danish National Health Registries.</p></li>
<li><p>To identify metabolites and biomarkers for liver health and inflammation mediating the association between genetically predicted (Fenland study, ADDITION-PRO, UK Biobank) and phenotypically measured (Fenland study) incretin and glucagon responses, and cardiometabolic outcomes; with a particular emphasis on direct and indirect markers of liver function.</p></li>
<li><p>To map dimensions of metabolic traits in conjunction with incretin and glucagon responses, in order to decipher their role in the context of deteriorated glucose metabolism and their relationship to long-term cardiometabolic health</p></li>
</ol>
</section>
<section id="background-and-significance-of-the-project" class="level2">
<h2 class="anchored" data-anchor-id="background-and-significance-of-the-project">Background and significance of the project</h2>
<p>Pre-diabetes is a complex state associated with increased risk of progression to type 2 diabetes and related complications<span class="citation" data-cites="wagner2021 tabák2012 birkenfeld2024"><sup><a href="#ref-wagner2021" role="doc-biblioref">1</a>–<a href="#ref-birkenfeld2024" role="doc-biblioref">3</a></sup></span>. However, not all people with pre-diabetes progress to diabetes; many persist in the prediabetic state or regress to normoglycemia over the course of various years<span class="citation" data-cites="tabák2012"><sup><a href="#ref-tabák2012" role="doc-biblioref">2</a></sup></span>. Remission to normoglycemia has been shown to reduce the risk of cardiovascular morbidity and mortality, however, it remains unclear to which extend if the remission is driven by lifestyle changes or by a heterogeneous low-risk subgroup<span class="citation" data-cites="vazquezarreola"><sup><a href="#ref-vazquezarreola" role="doc-biblioref">4</a></sup></span>.</p>
<p>Recently the idea of metabolic heterogeneity in type 2 diabetes has been extended to pre-diabetes, and traits that distinguish individuals most likely to progress to diabetes, such as insulin-resistant fatty liver and visceral adiposity-related renal dysfunction, have been identified<span class="citation" data-cites="wagner2021"><sup><a href="#ref-wagner2021" role="doc-biblioref">1</a></sup></span>. In recent years, incretins (Glucagon Like Peptide 1 [GLP 1] and Gastric Inhibitory Polypeptide [GIP]) and the hormone glucagon have gained increasing attention as targets in therapeutics for prevention of type 2 diabetes and diabetes-related complications<span class="citation" data-cites="kahn2024 jastreboffaniam.2025 jastreboffaniam.2023 sanyal2024 lincoffa.michael2023"><sup><a href="#ref-kahn2024" role="doc-biblioref">5</a>–<a href="#ref-lincoffa.michael2023" role="doc-biblioref">9</a></sup></span>. GLP-1 and GIP enhance post-meal insulin secretion, supporting efficient nutrient handling<span class="citation" data-cites="nauck1986"><sup><a href="#ref-nauck1986" role="doc-biblioref">10</a></sup></span>. Glucagon serves as a counter-regulatory hormone to insulin and maintains glucose balance both during fasting and following glucose intake<span class="citation" data-cites="hædersdal2023"><sup><a href="#ref-hædersdal2023" role="doc-biblioref">11</a></sup></span>. However, in the context of heterogeneity in pre-diabetes, the role of natural incretin and glucagon responses for the progression to type 2 diabetes and diabetes-related complications remains unexplored. Prior research in the Danish ADDITION-PRO study showed that, individuals with pre-diabetes and type 2 diabetes exhibited up to 25% lower natural GLP-1 responses to oral glucose<span class="citation" data-cites="færch2015"><sup><a href="#ref-færch2015" role="doc-biblioref">12</a></sup></span>. Additionally, impaired glucagon suppression and elevated fasting glucagon levels were observed in individuals with insulin resistance and early glucose dysregulation<span class="citation" data-cites="færch2016"><sup><a href="#ref-færch2016" role="doc-biblioref">13</a></sup></span>. A limitation of the ADDITION-PRO cohort is the selection of individuals at high risk of diabetes based on diabetes-specific risk scores, which limits the generalizability of the findings to the broader populations.</p>
<p>The Fenland Study represents a pioneering population-based cohort that has substantially advanced the precision of identifying individuals at high risk of type 2 diabetes<span class="citation" data-cites="carrasco-zanini2022"><sup><a href="#ref-carrasco-zanini2022" role="doc-biblioref">14</a></sup></span> and elucidating causal pathways underlying cardiometabolic disease [<span class="citation" data-cites="desilva2019"><sup><a href="#ref-desilva2019" role="doc-biblioref">15</a></sup></span>]<span class="citation" data-cites="wittemans2019"><sup><a href="#ref-wittemans2019" role="doc-biblioref">16</a></sup></span>[<span class="citation" data-cites="lotta2021"><sup><a href="#ref-lotta2021" role="doc-biblioref">17</a></sup></span>]<span class="citation" data-cites="agrawal2024"><sup><a href="#ref-agrawal2024" role="doc-biblioref">18</a></sup></span>. Leveraging extensive phenotypic and genetic data, the study has established causal relationships between liver function markers, alanine aminotransferase (ALT), aspartate aminotransferase (AST), and alkaline phosphatase (ALP), and type 2 diabetes<span class="citation" data-cites="desilva2019"><sup><a href="#ref-desilva2019" role="doc-biblioref">15</a></sup></span>, as well as between Dual Energy X-ray Absorptiometry (DEXA) derived adiposity traits, fat-mass ratios, and metabolic dysfunction-associated liver disease<span class="citation" data-cites="agrawal2024"><sup><a href="#ref-agrawal2024" role="doc-biblioref">18</a></sup></span>. Moreover, Fenland data have uncovered mechanistic links between interleukin-6–mediated inflammation and type 2 diabetes<span class="citation" data-cites="bowker2020"><sup><a href="#ref-bowker2020" role="doc-biblioref">19</a></sup></span>, highlighting the interplay between inflammatory and metabolic pathways. The cohort’s precise characterization of adiposity, inflammation, and hepatic health, combined with detailed assessments of incretin, glucagon, insulin, and glucose dynamics in a general population, provides an unparalleled resource for investigating the constellation of metabolic traits and their long-term impact on cardiometabolic health. Their cohort provide an unique timing for studying contellations of metabolic traits relevant to progression to cardiometablic disease and how the play in conjuctions to glucose and hormones.</p>
<p>This project is grounded in the hypothesis that dysregulated incretin secretion and inadequate glucagon suppression, together with insulin resistance, accelerate the progression of metabolic dysfunction by activating hepatic and inflammatory pathways in individuals with pre-diabetes.</p>
<p><a href="https://dp-next.github.io/wp3.html">Work Package 3 (Heterogeneity) in the new DP-Next</a></p>
<section id="clinical-applicability" class="level3">
<h3 class="anchored" data-anchor-id="clinical-applicability">Clinical applicability</h3>
<p>(Note from NNF: If you have an active grant from the Foundation, this may be taken into consideration in the evaluation of your application for a new grant. In general, it is recommended that the main applicant has delivered results on the active grant(s) before submission of a new application to the Foundation. If you apply while having an active grant from the Foundation, you must describe how the project you propose in this application is different from and/or coherent with the project(s) already funded and briefly describe the progress of the already funded project(s). This information should be included in the Project Description)</p>
<p>The Inter-SUSTAIN project supports the development of a precision approach to pre-diabetes by identifying a set of easily obtainable biomarkers that optimally distinguish individuals with a high probability of stable pre-diabetes or remission from those at greatest risk of progressing to diabetes. This work aligns with the objectives of the Novo Nordisk Foundation-funded Steno National grant, specifically <a href="https://dp-next.github.io/wp3.html">Work Package 3 (Heterogeneity) in the new DP-Next</a>, which aims to develop new strategies for diabetes prevention<span class="citation" data-cites="witte2025"><sup><a href="#ref-witte2025" role="doc-biblioref">20</a></sup></span>. The project will be in close connection with the DP-Next team and it is expected that the findings from the Fenland study and ADDITION-PRO will inform the design and structure of the new DP-Next cohort.</p>
<p>The Fenland study offers a unique opportunity through long-term follow-up data to support these efforts. It enables the identification of phenotypic and biological metabolic traits that characterize individuals at high risk of developing type 2 diabetes and related complications, including macrovascular and microvascular conditions, as well as increased mortality. Furthermore, these traits can be linked to longitudinal changes in HbA1c, lipid profiles, and eGFR, allowing for the identification of individuals with accelerated progression toward cardiometabolic complications. These insights support the timely identification of individuals who may benefit from early interventions, such as intensive lifestyle modifications or specific medications.</p>
</section>
</section>
<section id="work-package-1-investigate-causal-relationship-of-neutral-release-of-incretin-and-glucagon-and-long-term-cardiometabolic-disease-and-identify-mediating-pathways." class="level2">
<h2 class="anchored" data-anchor-id="work-package-1-investigate-causal-relationship-of-neutral-release-of-incretin-and-glucagon-and-long-term-cardiometabolic-disease-and-identify-mediating-pathways.">Work Package 1: Investigate causal relationship of neutral release of incretin and glucagon and long-term cardiometabolic disease and identify mediating pathways.</h2>
<p>Hypothesis for Work Package 1: Natural incretin and glucagon responses play a causal role as drivers of the risk of progression from pre-diabetes to type 2 diabetes and diabetes-related complications. In this pathway, biomarker and metabolites related to liver health play a mediating role.</p>
<p>Natural incretin and glucagon reposes in epidemiological studies have been mostly investigated cross-sectionally. Longitudinally, a higher GLP-1 response during OGTT was associated with a steeper decrease in fasting glucose over time<span class="citation" data-cites="koopman2018"><sup><a href="#ref-koopman2018" role="doc-biblioref">21</a></sup></span>; however, with a limited sample size (n=121). The ADDITION PRO cohort includes 1657 individuals at high risk of diabetes who underwent measurement of natural incretin and glucagon responses during a 3-point (0, 30, 120 min) oral glucose tolerance test (OGTT) in 2009-2010, i.e.&nbsp;before the introduction of incretin based therapies<span class="citation" data-cites="færch2015 færch2016 johansen2012"><sup><a href="#ref-færch2015" role="doc-biblioref">12</a>,<a href="#ref-færch2016" role="doc-biblioref">13</a>,<a href="#ref-johansen2012" role="doc-biblioref">22</a></sup></span>. This unique timing provides an opportunity to prospectively analyze the associations between individuals’s natural hormonal responses to glucose and progression to diabetes, regression to normoglycemia, or risk of diabetes-related complications.</p>
<p>Preliminary findings from ADDITION PRO indicate that per SD higher glucagon (incidence rate ratio IRR: 1.38 CI: 1.15 to 1.67) and GLP-1 responses (IRR: 1.34 CI: 1.05 to 1.71) at 120 minutes during the OGTT are associated with incident heart failure over an 11-year follow-up period.</p>
<p>Genetic variants, clinical biomarkers and metabolites serve as valuable indicators of individual predisposition for specific phenotypes (genes) and mediating pathways (metabolites and clinical biomarkers) that link phenotypes to disease outcomes. Genetic variants can strengthen causal inference by serving as instrumental variables in Mendelian randomization studies<span class="citation" data-cites="daveysmith2014"><sup><a href="#ref-daveysmith2014" role="doc-biblioref">23</a></sup></span>. Identification of metabolic pathways that mediate the effect of genetically determined phenotypes on the incidence of diseases such as type 2 diabetes helps us understand the causal biological mechanisms linking traits to disease. This understanding in turn supports the development of potential new targeted interventions based on metabolic pathways.</p>
<p>Evaluating the risk associated with incretin and glucagon responses to a glucose load in isolation may be limited without considering concurrent insulin and glucose regulation. To address this, in WP1 principal component analysis will be applied to extract the strongest sources of variance in the concurrent OGTT response across all 4 hormones and glucose.</p>
<section id="research-design-and-methods" class="level3">
<h3 class="anchored" data-anchor-id="research-design-and-methods">Research design and methods</h3>
<p>To quantify the causal relationship between incretins and glucagon and cardiometabolic disease, we will first apply traditional epidemiological time-to-event analysis methods (Poisson and Cox regression) in Fenland study, with adjustments informed by pathways outlined in directed acyclic graphs (DAGs). To strengthen causal inference and account for unmeasured confounding, Mendelian randomization will be conducted using pathway specific polygenic risk scores (PRS) as instruments for incretins and glucagon<span class="citation" data-cites="daveysmith2014"><sup><a href="#ref-daveysmith2014" role="doc-biblioref">23</a></sup></span>, leveraging data from ADDITION-PRO, the UK Biobank and Fenland Study<span class="citation" data-cites="mrcepidemiologyunit"><sup><a href="#ref-mrcepidemiologyunit" role="doc-biblioref">24</a></sup></span>.</p>
<p>To sequence mediating pathways and identify key metabolites linking incretin and glucagon responses to cardiometabolic disease, we will apply a structural causal algorithm (NetCoupler)<span class="citation" data-cites="johnston2020"><sup><a href="#ref-johnston2020" role="doc-biblioref">25</a></sup></span> to identify sequential causal pathways involving metabolites. Based on the identified metabolites, structured causal mediation analyses<span class="citation" data-cites="vanderweele2015"><sup><a href="#ref-vanderweele2015" role="doc-biblioref">26</a></sup></span> will be conducted to assess and quantify the direct and indirect effects of individual metabolites seen as most likely to be causal mediators in NetCoupler on cardiometabolic outcomes.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><embed src="figures/DDEA_Proposal.pdf" class="img-fluid" style="width:8in"></p>
<figcaption class="figure-caption"><strong>Figure 1: Structured causal analysis framework of incretin and glucagon responses and their association with the risk of progression to cardiometabolic disease</strong></figcaption>
</figure>
</div>
<p><em>trash can</em> and in a subgroup of around 800 participants, still B-mode ultrasound images of the liver, collected with a standardized protocol<span class="citation" data-cites="johansen2012"><sup><a href="#ref-johansen2012" role="doc-biblioref">22</a></sup></span>. WP3 will apply novel AI-based image processing and analysis methods to extract features and cluster images.</p>
</section>
</section>
<section id="work-package-2-to-map-dimensions-of-metabolic-traits-in-conjunction-with-incretin-and-glucagon-responses-in-order-to-decipher-their-role-in-the-context-of-deteriorated-glucose-metabolism." class="level2">
<h2 class="anchored" data-anchor-id="work-package-2-to-map-dimensions-of-metabolic-traits-in-conjunction-with-incretin-and-glucagon-responses-in-order-to-decipher-their-role-in-the-context-of-deteriorated-glucose-metabolism.">Work Package 2: To map dimensions of metabolic traits in conjunction with incretin and glucagon responses, in order to decipher their role in the context of deteriorated glucose metabolism.</h2>
<p>Hypothesis for Work Package 2: natural incretin and glucagon responses may exert protective, compensatory effects under conditions of obesity, hyperglycemia, insulin-resistance, and loss of beta-cell function. We further hypothesize that liver health and low-grade infalmmation may act as an effect modifier in the associations between incretin-glucagon responses and metabolic outcomes.</p>
<p>Obesity and insulin resistance have long been regarded as central components in the development of type 2 diabetes. Recent clustering analyses have identified liver fat as a main determinant heterogeneity among people with pre-diabetes with regard to their risk of progression to type 2 diabetes<span class="citation" data-cites="wagner2021"><sup><a href="#ref-wagner2021" role="doc-biblioref">1</a></sup></span>. Reduced liver function interacts with insulin resistance, glucose and glucagon responses, and is accompanied by low grade inflammation and morphological changes to the liver, particularly MAFLD (Metabolically Associated Fatty Liver Disease)<span class="citation" data-cites="winther-sørensen2020 stefan2025 hædersdal2023"><sup><a href="#ref-hædersdal2023" role="doc-biblioref">11</a>,<a href="#ref-winther-sørensen2020" role="doc-biblioref">27</a>,<a href="#ref-stefan2025" role="doc-biblioref">28</a></sup></span>. The biological actions of GLP-1 and GIP include reducing hepatic inflammation<span class="citation" data-cites="hammoud2023"><sup><a href="#ref-hammoud2023" role="doc-biblioref">29</a></sup></span>. Liver function therefore appears to be a central pathway modulating diabetes risk and interacting with or mediating the impact of natural incretin and glucagon responses. The Fenland study includes baseline measurements of γ-glutamyl transferase and alanine transaminase, and 11,559 participants had DEXA<span class="citation" data-cites="powell2020"><sup><a href="#ref-powell2020" role="doc-biblioref">30</a></sup></span> performed allowing to estimate regional fatmass inclunding liver fat. In conjunction with liver function biomarkers, this will enable the calculation of a general indication of liver health.</p>
<p>We will further integrate dimensions of metabolically relevant low-grade inflammation by integrating markers of macrophage activation (soluble CD163), adiponectin, and C-reactive protein<span class="citation" data-cites="deichgræber2016"><sup><a href="#ref-deichgræber2016" role="doc-biblioref">31</a></sup></span> in our multidimensional characterization of liver health. In addition, fasting branched-chain amino acids (BCAA)<span class="citation" data-cites="mahendran2017"><sup><a href="#ref-mahendran2017" role="doc-biblioref">32</a></sup></span>and lipid-related metabolites during the OGTT<span class="citation" data-cites="buckley2017"><sup><a href="#ref-buckley2017" role="doc-biblioref">33</a></sup></span> were obtained in ADDITION-PRO. These data allow us to characterize aspects of metabolic function, such as liver fat accumulation<span class="citation" data-cites="gnatiucfriedrichs2023"><sup><a href="#ref-gnatiucfriedrichs2023" role="doc-biblioref">34</a></sup></span>and tissue-specific insulin resistance<span class="citation" data-cites="vogelzangs2020 beyene2020"><sup><a href="#ref-vogelzangs2020" role="doc-biblioref">35</a>,<a href="#ref-beyene2020" role="doc-biblioref">36</a></sup></span> .</p>
<p>Based on phenotypes, metabolites, and genotypes, we want to identify metabolic traits that, in conjunction with incretin and glucagon responses, contribute to cardiometabolic risk. Machine learning–based dimensionality reduction techniques help identify and characterize distinct constellations of metabolic traits across individuals. From phenotypic clusters, we want to map dimensions of metabolic function that either provide compensatory protection against or contribute to cardiometabolic disease. Then, we will add dimensions of metabolite-based profiles to the existing clusters to enhance their characterization.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><embed src="figures/fig2_pca.pdf" class="img-fluid" style="width:8in"></p>
<figcaption class="figure-caption">Figure 2: Illustration of dimensionality reduction of metabolic traits into metabolic clusters</figcaption>
</figure>
</div>
<p>To support phenotypic clustering, genetic data will be incorporated through partitioned polygenic risk scores (PRS) for type 2 diabetes. These scores reflect underlying traits such as glucagon and incretin, inflammation, beta-cell function, obesity, liver fat, and insulin resistance. This approach enables the identification of key phenotypes by genetic risk scores that drive cardiometabolic disease.</p>
<section id="research-design-and-methods-1" class="level3">
<h3 class="anchored" data-anchor-id="research-design-and-methods-1">Research design and methods</h3>
<p>We use data from previous work in The Fenland study and ADDITION-PRO, including genotypes, metabolomic profiles, and markers of metabolism and inflammation<span class="citation" data-cites="mahendran2017 buckley2017 deichgræber2016 johansen2012 færch2015 færch2016 madsen2024"><sup><a href="#ref-færch2015" role="doc-biblioref">12</a>,<a href="#ref-færch2016" role="doc-biblioref">13</a>,<a href="#ref-johansen2012" role="doc-biblioref">22</a>,<a href="#ref-deichgræber2016" role="doc-biblioref">31</a>–<a href="#ref-buckley2017" role="doc-biblioref">33</a>,<a href="#ref-madsen2024" role="doc-biblioref">37</a></sup></span>.</p>
<p>Combined with liver function biomarkers, this approach will enable us to generate a general, multidimensional indication of liver health.</p>
<p>We will apply principal component analysis to integrate variables and identify a restricted number of metabolic dimensions that reflect the interplay between incretin and glucagon responses along with concurrent glucose-insulin dynamics during the OGTT<span class="citation" data-cites="zhou2024"><sup><a href="#ref-zhou2024" role="doc-biblioref">38</a></sup></span>. We will further add dimensions of obesity, insulin resistance, inflammation, liver health markers, subsequently examine these dimensions in relation to diabetes risk<span class="citation" data-cites="healy2024"><sup><a href="#ref-healy2024" role="doc-biblioref">39</a></sup></span>. Finally, metabolite-based profiles and partitioned PRS will be analyzed separately in relation to cardiometabolic outcomes and subsequently incorporated to enrich the dimensional metabolic profiling. All components will be investigated prospectively in relation to cardiometabolic outcomes.</p>
<p><em>trash can</em> and in a subgroup of around 800 participants, still B-mode ultrasound images of the liver, collected with a standardized protocol<span class="citation" data-cites="johansen2012"><sup><a href="#ref-johansen2012" role="doc-biblioref">22</a></sup></span>. WP3 will apply novel AI-based image processing and analysis methods to extract features and cluster images. We will leverage recent advances in AI-based image analysis by applying transfer learning approaches and open-source feature extraction and image classification algorithms to ultrasound liver images<span class="citation" data-cites="cohn2021"><sup><a href="#ref-cohn2021" role="doc-biblioref">40</a></sup></span>.</p>
</section>
</section>
<section id="work-package-3-to-predict-incretin-and-glucagon-responses-in-conjuction-with-metabolic-traits-based-clinical-and-biological-traits" class="level2">
<h2 class="anchored" data-anchor-id="work-package-3-to-predict-incretin-and-glucagon-responses-in-conjuction-with-metabolic-traits-based-clinical-and-biological-traits">Work Package 3: To predict incretin and glucagon responses in conjuction with metabolic traits based clinical and biological traits</h2>
<p>Hypothesis for Work Package 3: population heterogeneity of incretin and glucagon responses in conjunction with metabolic traits can be predicted to a clinically useful degree based on clinical, genetic and metabolic biomarkers, and these predictions can stratify individuals by their risk of progression to type 2 diabetes and related complications.</p>
<p>Given the fact that it is impractical to assess a full incretin and glucagon response in clinical practice, WP3 aims to establish a practical set of biomarkers that jointly have the capacity to characterize incretin and glucagon responses. Analyses in the Fenland study will be carried out across three levels: (1) traditional clinical characteristics, (2) metabolomics profiles, and (3) genomic data.The utility of these predicted traits will be investigated in relation to cardiometabolic outcomes in large population-based cohorts that include the same biomarkers but have not measured incretin and glucagon responses, such as the UK Biobank, as well as using available markers in Danish registries.</p>
<section id="research-design-and-methods-2" class="level3">
<h3 class="anchored" data-anchor-id="research-design-and-methods-2">Research design and methods</h3>
<p>To predict incretin and glucagon responses based on clinical and metabolite data, we will apply a two-layered modeling approach in the Fenland study. Clinical and metabolic profiles will be used separately and in combination to predict responses. For variable selection in both layers, we will apply two strategies: restricting variance and using linear LASSO regression to shrink and select the most predictive variables. To support this, we will employ machine learning models to capture undefined interactions and identify important predictors based on variable importance<span class="citation" data-cites="dietrich2016"><sup><a href="#ref-dietrich2016" role="doc-biblioref">41</a></sup></span>. Our approach follows established principles in machine learning and statistical modeling, adhering to standardized practices for prediction, reporting, and validation<span class="citation" data-cites="collins2024 lopez-ayala2025 collins2024a riley2024"><sup><a href="#ref-collins2024" role="doc-biblioref">42</a>–<a href="#ref-riley2024" role="doc-biblioref">45</a></sup></span>.</p>
<p>As part of the collaboration with the MRC Epidemiology Unit and Novo Nordisk Foundation Center for Basic Metabolic Research, current efforts are focused on developing PRS for GLP-1, GIP, and glucagon. These PRSs will be incorporated into the aforementioned predictive layers as composite predictors of incretin and glucagon responses. PRS for other relevant traits such as obesity, insulin resistance, beta-cell function, liver function and low-grade inflammation will also be considered where relevant.</p>
</section>
</section>
<section id="research-design-and-methods-3" class="level2">
<h2 class="anchored" data-anchor-id="research-design-and-methods-3">Research design and methods</h2>
<table class="table">
<colgroup>
<col style="width: 18%">
<col style="width: 42%">
<col style="width: 39%">
</colgroup>
<thead>
<tr class="header">
<th></th>
<th><strong>Fenland study</strong></th>
<th><strong>ADDITION-PRO Study</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Baseline data collection period</td>
<td>2005-2015 (Phase 1-3)</td>
<td>2009–2011</td>
</tr>
<tr class="even">
<td>Included participants</td>
<td>UK general population aged 30-55 years</td>
<td>Individuals at high risk of diabetes</td>
</tr>
<tr class="odd">
<td>Number of participants</td>
<td>12,435</td>
<td>2,082</td>
</tr>
<tr class="even">
<td>Measure from OGTT (t= 0, 30, 120 min):</td>
<td>GLP-1, GIP, glucagon, glucose and insulin</td>
<td>GLP-1, GIP, glucagon, glucose and insulin</td>
</tr>
<tr class="odd">
<td>Other relevant metabolic measures</td>
<td>DEXA-scan</td>
<td><p>Liver function:</p>
<ul>
<li>Ultrasound liver images (still B-mode images with liver protocol)</li>
<li>GGT, Alanine aminotransferase</li>
</ul>
<p>Inflammation:</p>
<ul>
<li>soluble CD163, adiponectin, and high sensitivity C-reactive protein</li>
</ul></td>
</tr>
<tr class="even">
<td>Assessment method of metabolites</td>
<td>LC electrospray ionization and flow-injection analysis tandem MS (ref), targeted metabolomics</td>
<td>Proton nuclear magnetic resonance spectroscopy, targeted metabolomics<span class="citation" data-cites="mahendran2017"><sup><a href="#ref-mahendran2017" role="doc-biblioref">32</a></sup></span></td>
</tr>
<tr class="odd">
<td>Sample tissue</td>
<td>Fasting plasma blood samples</td>
<td>Both fasting and during the OGTT<span class="citation" data-cites="buckley2017 mahendran2017"><sup><a href="#ref-mahendran2017" role="doc-biblioref">32</a>,<a href="#ref-buckley2017" role="doc-biblioref">33</a></sup></span></td>
</tr>
<tr class="even">
<td>Number of metabolites</td>
<td>175 (acylcarnitines, amines, sphingolipids and phospholipids)</td>
<td>231 lipid-related and 3 BCAA (fasting isoleucine, leucine and valine levels)</td>
</tr>
<tr class="odd">
<td>Genotyping</td>
<td>Yes</td>
<td>Yes</td>
</tr>
<tr class="even">
<td>Follow-up period</td>
<td>NA</td>
<td>13-years</td>
</tr>
<tr class="odd">
<td>Outcome</td>
<td><p>Metabolic-related outcomes:</p>
<ul>
<li>Progression to type 2 diabetes</li>
</ul>
<p>Macrovascular complications:</p>
<ul>
<li><p>Ischemic-related cardiovascular disease</p></li>
<li><p>Heart failure</p></li>
</ul>
<p>Microvascular complications:</p>
<ul>
<li>Chronic kidney disease</li>
</ul></td>
<td><p>Metabolic-related outcomes:</p>
<ul>
<li><p>Progression to type 2 diabetes</p></li>
<li><p>Regression to normoglycemia</p></li>
<li><p>HbA1c, lipids and eGFR trajectories</p></li>
</ul>
<p>Macrovascular complications:</p>
<ul>
<li><p>Ischemic-related cardiovascular disease</p></li>
<li><p>Heart failure</p></li>
</ul>
<p>Microvascular complications:</p>
<ul>
<li><p>Chronic kidney disease</p></li>
<li><p>Neuropathy</p></li>
</ul></td>
</tr>
</tbody>
</table>
<p>To extend findings and predicted characterizations in larger population-based cohorts, will use ADDITION-PRO Study and UK Biobank<span class="citation" data-cites="bycroft2018"><sup><a href="#ref-bycroft2018" role="doc-biblioref">46</a></sup></span> and as validation cohorts.</p>
</section>
<section id="collaboration" class="level2">
<h2 class="anchored" data-anchor-id="collaboration">Collaboration</h2>
<p>The synergy between this project and the MRC Epidemiology Unit lies in the integration of deep-phenotyped metabolic data with large-scale population health surveillance. While my work in Denmark leverages the specialized ADDITION-PRO cohort, the MRC Epidemiology Unit provides an unparalleled environment for precision epidemiology. By combining the Unit’s global leadership in studying the genetic and environmental determinants of obesity and type 2 diabetes with my focus on natural incretin and glucagon responses, we create a powerful framework to move from observational data to causal understanding. This collaboration allows for the cross-pollination of Danish clinical depth and the UK’s extensive population-based datasets, ensuring that findings regarding metabolic drivers are both biologically robust and representative of the broader population.</p>
<p>The MRC Epidemiology Unit, specifically through the mentorship of experts in the Fenland Study including Professor Nick Wareham and Professor Simon Griffin, will play a critical role as the primary international host and scientific advisor. Their role is three-fold:</p>
<ol type="1">
<li><p>Data Provision and Oversight: Providing access to the Fenland Study (n=12,435), which is uniquely suited for this project due to its objective measures of metabolic function and long-term follow-up.</p></li>
<li><p>Methodological Mentorship: Facilitating training in causal inference and Mendelian Randomization, specifically using genetic instruments to determine the causal role of incretin and glucagon in cardiometabolic outcomes.</p></li>
<li><p>Validation Expertise: Assisting in the harmonisation of data between the Fenland Study, UK Biobank, and ADDITION-PRO to ensure that the identified metabolic profiles and mediating pathways (such as liver health and inflammation) are validated across different geographical and demographic contexts.</p></li>
</ol>
<p>The synergy with the MRC Epidemiology Unit will be an ongoing institutional bridge. Our research is strategically aligned with the Unit’s mission to improve population health through better risk stratification.</p>
<p>Continuous Methodological Alignment: We will maintain regular scientific exchange through joint virtual meetings to align the analytical protocols used in the Fenland Study with the DP-Next (Work Package 3) framework.</p>
<p>Future Frameworks: This collaboration serves as a pilot for long-term data integration between Danish and UK cohorts, establishing a permanent channel for investigating the heterogeneity of type 2 diabetes and refining precision prevention strategies across borders.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><embed src="figures/gant_diagram.pdf" class="img-fluid" style="width:8in"></p>
<figcaption class="figure-caption"><strong>Figure 3: Gantt chart</strong></figcaption>
</figure>
</div>
</section>
</section>
<section id="lay-project-description" class="level1 unnumbered">


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">LAY PROJECT DESCRIPTION</h2><div id="refs" class="references csl-bib-body" role="list">
<div id="ref-wagner2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">1 </div><div class="csl-right-inline">Wagner R, Heni M, Tabák AG, <em>et al.</em> <a href="https://doi.org/10.1038/s41591-020-1116-9">Pathophysiology-based subphenotyping of individuals at elevated risk for type 2 diabetes</a>. <em>Nature Medicine</em> 2021; <strong>27</strong>: 49–57.</div>
</div>
<div id="ref-tabák2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">2 </div><div class="csl-right-inline">Tabák AG, Herder C, Rathmann W, Brunner EJ, Kivimäki M. <a href="https://doi.org/10.1016/S0140-6736(12)60283-9">Prediabetes: A high-risk state for diabetes development</a>. <em>The Lancet</em> 2012; <strong>379</strong>: 2279–90.</div>
</div>
<div id="ref-birkenfeld2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">3 </div><div class="csl-right-inline">Birkenfeld AL, Franks PW, Mohan V. <a href="https://doi.org/10.1161/CIRCULATIONAHA.124.070463">Precision medicine in people at risk for diabetes and atherosclerotic cardiovascular disease: A fresh perspective on prevention</a>. <em>Circulation</em> 2024; <strong>150</strong>: 1910–2.</div>
</div>
<div id="ref-vazquezarreola" class="csl-entry" role="listitem">
<div class="csl-left-margin">4 </div><div class="csl-right-inline">Vazquez Arreola E, Gong Q, Hanson RL, <em>et al.</em> Prediabetes remission and cardiovascular morbidity and mortality: Post-hoc analyses from the diabetes prevention program outcome study and the DaQing diabetes prevention outcome study. <em>The Lancet Diabetes &amp; Endocrinology</em> DOI:<a href="https://doi.org/10.1016/S2213-8587(25)00295-5">10.1016/S2213-8587(25)00295-5</a>.</div>
</div>
<div id="ref-kahn2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">5 </div><div class="csl-right-inline">Kahn SE, Deanfield JE, Jeppesen OK, <em>et al.</em> <a href="https://doi.org/10.2337/dc24-0491">Effect of semaglutide on regression and progression of glycemia in people with overweight or obesity but without diabetes in the SELECT trial</a>. <em>Diabetes Care</em> 2024; <strong>47</strong>: 1350–9.</div>
</div>
<div id="ref-jastreboffaniam.2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">6 </div><div class="csl-right-inline">Jastreboff Ania M., Roux Carel W. le, Stefanski Adam, <em>et al.</em> <a href="https://doi.org/10.1056/NEJMoa2410819">Tirzepatide for obesity treatment and diabetes prevention</a>. <em>New England Journal of Medicine</em> 2025; <strong>392</strong>: 958–71.</div>
</div>
<div id="ref-jastreboffaniam.2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">7 </div><div class="csl-right-inline">Jastreboff Ania M., Kaplan Lee M., Frías Juan P., <em>et al.</em> <a href="https://doi.org/10.1056/NEJMoa2301972">Triple<span></span>hormone-receptor agonist retatrutide for obesity <span></span> a phase 2 trial</a>. <em>New England Journal of Medicine</em> 2023; <strong>389</strong>: 514–26.</div>
</div>
<div id="ref-sanyal2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">8 </div><div class="csl-right-inline">Sanyal AJ, Kaplan LM, Frias JP, <em>et al.</em> <a href="https://doi.org/10.1038/s41591-024-03018-2">Triple hormone receptor agonist retatrutide for metabolic dysfunction-associated steatotic liver disease: A randomized phase 2a trial</a>. <em>Nature Medicine</em> 2024; <strong>30</strong>: 2037–48.</div>
</div>
<div id="ref-lincoffa.michael2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">9 </div><div class="csl-right-inline">Lincoff A. Michael, Brown-Frandsen Kirstine, Colhoun Helen M., <em>et al.</em> <a href="https://doi.org/10.1056/NEJMoa2307563">Semaglutide and cardiovascular outcomes in obesity without diabetes</a>. <em>New England Journal of Medicine</em> 2023; <strong>389</strong>: 2221–32.</div>
</div>
<div id="ref-nauck1986" class="csl-entry" role="listitem">
<div class="csl-left-margin">10 </div><div class="csl-right-inline">NAUCK MA, HOMBERGER E, SIEGEL EG, <em>et al.</em> <a href="https://doi.org/10.1210/jcem-63-2-492">Incretin effects of increasing glucose loads in man calculated from venous insulin and c-peptide responses*</a>. <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 1986; <strong>63</strong>: 492–8.</div>
</div>
<div id="ref-hædersdal2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">11 </div><div class="csl-right-inline">Hædersdal S, Andersen A, Knop FK, Vilsbøll T. <a href="https://doi.org/10.1038/s41574-023-00817-4">Revisiting the role of glucagon in health, diabetes mellitus and other metabolic diseases</a>. <em>Nature Reviews Endocrinology</em> 2023; <strong>19</strong>: 321–35.</div>
</div>
<div id="ref-færch2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">12 </div><div class="csl-right-inline">Færch K, Torekov SS, Vistisen D, <em>et al.</em> <a href="https://doi.org/10.2337/db14-1751">GLP-1 response to oral glucose is reduced in prediabetes, screen-detected type 2 diabetes, and obesity and influenced by sex: The ADDITION-PRO study</a>. <em>Diabetes</em> 2015; <strong>64</strong>: 2513–25.</div>
</div>
<div id="ref-færch2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">13 </div><div class="csl-right-inline">Færch K, Vistisen D, Pacini G, <em>et al.</em> <a href="https://doi.org/10.2337/db16-0240">Insulin resistance is accompanied by increased fasting glucagon and delayed glucagon suppression in individuals with normal and impaired glucose regulation</a>. <em>Diabetes</em> 2016; <strong>65</strong>: 3473–81.</div>
</div>
<div id="ref-carrasco-zanini2022" class="csl-entry" role="listitem">
<div class="csl-left-margin">14 </div><div class="csl-right-inline">Carrasco-Zanini J, Pietzner M, Lindbohm JV, <em>et al.</em> <a href="https://doi.org/10.1038/s41591-022-02055-z">Proteomic signatures for identification of impaired glucose tolerance</a>. <em>Nature Medicine</em> 2022; <strong>28</strong>: 2293–300.</div>
</div>
<div id="ref-desilva2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">15 </div><div class="csl-right-inline">De Silva NMG, Borges MC, Hingorani AD, <em>et al.</em> <a href="https://doi.org/10.2337/db18-1048">Liver function and risk of type 2 diabetes: Bidirectional mendelian randomization study</a>. <em>Diabetes</em> 2019; <strong>68</strong>: 1681–91.</div>
</div>
<div id="ref-wittemans2019" class="csl-entry" role="listitem">
<div class="csl-left-margin">16 </div><div class="csl-right-inline">Wittemans LBL, Lotta LA, Oliver-Williams C, <em>et al.</em> <a href="https://doi.org/10.1038/s41467-019-08936-1">Assessing the causal association of glycine with risk of cardio-metabolic diseases</a>. <em>Nature Communications</em> 2019; <strong>10</strong>: 1060.</div>
</div>
<div id="ref-lotta2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">17 </div><div class="csl-right-inline">Lotta LA, Pietzner M, Stewart ID, <em>et al.</em> <a href="https://doi.org/10.1038/s41588-020-00751-5">A cross-platform approach identifies genetic regulators of human metabolism and health</a>. <em>Nature Genetics</em> 2021; <strong>53</strong>: 54–64.</div>
</div>
<div id="ref-agrawal2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">18 </div><div class="csl-right-inline">Agrawal S, Luan J, Cummings BB, Weiss EJ, Wareham NJ, Khera AV. <a href="https://doi.org/10.2337/db23-0575">Relationship of fat mass ratio, a biomarker for lipodystrophy, with cardiometabolic traits</a>. <em>Diabetes</em> 2024; <strong>73</strong>: 1099–111.</div>
</div>
<div id="ref-bowker2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">19 </div><div class="csl-right-inline">Bowker N, Shah RL, Sharp SJ, <em>et al.</em> <a href="https://doi.org/10.1016/j.ebiom.2020.103062">Meta-analysis investigating the role of interleukin-6 mediated inflammation in type 2 diabetes</a>. <em>EBioMedicine</em> 2020; <strong>61</strong>: 103062.</div>
</div>
<div id="ref-witte2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">20 </div><div class="csl-right-inline">Witte DR, Johnston L, Røikjer J, Rasmussen N, Juhl CB. Work package 3: heterogeneity. 2025; published online Aug 19. <a href="https://dp-next.github.io/wp3.html">https://dp-next.github.io/wp3.html</a>.</div>
</div>
<div id="ref-koopman2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">21 </div><div class="csl-right-inline">Koopman ADM, Rutters F, Rauh SP, <em>et al.</em> <a href="https://doi.org/10.1371/journal.pone.0191114">Incretin responses to oral glucose and mixed meal tests and changes in fasting glucose levels during 7 years of follow-up: The hoorn meal study</a>. <em>PLOS ONE</em> 2018; <strong>13</strong>: e0191114.</div>
</div>
<div id="ref-johansen2012" class="csl-entry" role="listitem">
<div class="csl-left-margin">22 </div><div class="csl-right-inline">Johansen NB, Hansen AL, Jensen TM, <em>et al.</em> <a href="https://doi.org/10.1186/1471-2458-12-1078">Protocol for ADDITION-PRO: a longitudinal cohort study of the cardiovascular experience of individuals at high risk for diabetes recruited from Danish primary care</a>. <em>BMC Public Health</em> 2012; <strong>12</strong>: 1078.</div>
</div>
<div id="ref-daveysmith2014" class="csl-entry" role="listitem">
<div class="csl-left-margin">23 </div><div class="csl-right-inline">Davey Smith G, Hemani G. <a href="https://doi.org/10.1093/hmg/ddu328">Mendelian randomization: Genetic anchors for causal inference in epidemiological studies</a>. <em>Human Molecular Genetics</em> 2014; <strong>23</strong>: R89–98.</div>
</div>
<div id="ref-mrcepidemiologyunit" class="csl-entry" role="listitem">
<div class="csl-left-margin">24 </div><div class="csl-right-inline">MRC Epidemiology Unit. Fenland study. <a href="https://studies.mrc-epid.cam.ac.uk/fenland">https://studies.mrc-epid.cam.ac.uk/fenland</a>.</div>
</div>
<div id="ref-johnston2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">25 </div><div class="csl-right-inline">Johnston L, Wittenbecher C. Inference of causal links between metabolomics and disease incidence. 2020. <a href="https://github.com/NetCoupler/NetCoupler">https://github.com/NetCoupler/NetCoupler</a>.</div>
</div>
<div id="ref-vanderweele2015" class="csl-entry" role="listitem">
<div class="csl-left-margin">26 </div><div class="csl-right-inline">VanderWeele T. Explanation in causal inference: Methods for mediation and interaction. Oxford University Press, 2015.</div>
</div>
<div id="ref-winther-sørensen2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">27 </div><div class="csl-right-inline">Winther-Sørensen M, Galsgaard KD, Santos A, <em>et al.</em> <a href="https://doi.org/10.1016/j.molmet.2020.101080">Glucagon acutely regulates hepatic amino acid catabolism and the effect may be disturbed by steatosis</a>. <em>Molecular Metabolism</em> 2020; <strong>42</strong>: 101080.</div>
</div>
<div id="ref-stefan2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">28 </div><div class="csl-right-inline">Stefan N, Yki-Järvinen H, Neuschwander-Tetri BA. <a href="https://doi.org/10.1016/S2213-8587(24)00318-8">Metabolic dysfunction-associated steatotic liver disease: Heterogeneous pathomechanisms and effectiveness of metabolism-based treatment</a>. <em>The Lancet Diabetes &amp; Endocrinology</em> 2025; <strong>13</strong>: 134–48.</div>
</div>
<div id="ref-hammoud2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">29 </div><div class="csl-right-inline">Hammoud R, Drucker DJ. <a href="https://doi.org/10.1038/s41574-022-00783-3">Beyond the pancreas: Contrasting cardiometabolic actions of GIP and GLP1</a>. <em>Nature Reviews Endocrinology</em> 2023; <strong>19</strong>: 201–16.</div>
</div>
<div id="ref-powell2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">30 </div><div class="csl-right-inline">Powell R, De Lucia Rolfe E, Day FR, <em>et al.</em> <a href="https://doi.org/10.1101/2020.12.16.20248330">Development and validation of total and regional body composition prediction equations from anthropometry and single frequency segmental bioelectrical impedance with DEXA</a>. <em>medRxiv</em> 2020; : 2020.12.16.20248330.</div>
</div>
<div id="ref-deichgræber2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">31 </div><div class="csl-right-inline">Deichgræber P, Witte DR, Møller HJ, <em>et al.</em> <a href="https://doi.org/10.1007/s00125-016-4075-4">Soluble CD163, adiponectin, c-reactive protein and progression of dysglycaemia in individuals at high risk of type 2 diabetes mellitus: The ADDITION-PRO cohort</a>. <em>Diabetologia</em> 2016; <strong>59</strong>: 2467–76.</div>
</div>
<div id="ref-mahendran2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">32 </div><div class="csl-right-inline">Mahendran Y, Jonsson A, Have CT, <em>et al.</em> <a href="https://doi.org/10.1007/s00125-017-4222-6">Genetic evidence of a causal effect of insulin resistance on branched-chain amino acid levels</a>. <em>Diabetologia</em> 2017; <strong>60</strong>: 873–8.</div>
</div>
<div id="ref-buckley2017" class="csl-entry" role="listitem">
<div class="csl-left-margin">33 </div><div class="csl-right-inline">Buckley MT, Racimo F, Allentoft ME, <em>et al.</em> <a href="https://doi.org/10.1093/molbev/msx103">Selection in europeans on fatty acid desaturases associated with dietary changes</a>. <em>Molecular Biology and Evolution</em> 2017; <strong>34</strong>: 1307–18.</div>
</div>
<div id="ref-gnatiucfriedrichs2023" class="csl-entry" role="listitem">
<div class="csl-left-margin">34 </div><div class="csl-right-inline">Gnatiuc Friedrichs L, Trichia E, Aguilar-Ramirez D, Preiss D. <a href="https://doi.org/10.1002/oby.23687">Metabolic profiling of MRI-measured liver fat in the UK biobank</a>. <em>Obesity</em> 2023; <strong>31</strong>: 1121–32.</div>
</div>
<div id="ref-vogelzangs2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">35 </div><div class="csl-right-inline">Vogelzangs N, Kallen CJH van der, Greevenbroek MMJ van, <em>et al.</em> <a href="https://doi.org/10.1038/s41366-020-0565-z">Metabolic profiling of tissue-specific insulin resistance in human obesity: Results from the diogenes study and the maastricht study</a>. <em>International Journal of Obesity</em> 2020; <strong>44</strong>: 1376–86.</div>
</div>
<div id="ref-beyene2020" class="csl-entry" role="listitem">
<div class="csl-left-margin">36 </div><div class="csl-right-inline">Beyene HB, Hamley S, Giles C, <em>et al.</em> <a href="https://doi.org/10.1210/clinem/dgaa054">Mapping the associations of the plasma lipidome with insulin resistance and response to an oral glucose tolerance test</a>. <em>The Journal of Clinical Endocrinology &amp; Metabolism</em> 2020; <strong>105</strong>: e1041–55.</div>
</div>
<div id="ref-madsen2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">37 </div><div class="csl-right-inline">Madsen AL, Bonàs-Guarch S, Gheibi S, <em>et al.</em> <a href="https://doi.org/10.1038/s42255-024-01140-6">Genetic architecture of oral glucose-stimulated insulin release provides biological insights into type 2 diabetes aetiology</a>. <em>Nature Metabolism</em> 2024; <strong>6</strong>: 1897–912.</div>
</div>
<div id="ref-zhou2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">38 </div><div class="csl-right-inline">Zhou Y, Chen H, Iao SI, <em>et al.</em> Fdapace: Functional data analysis and empirical dynamics. 2024 <a href="https://CRAN.R-project.org/package=fdapace">https://CRAN.R-project.org/package=fdapace</a>.</div>
</div>
<div id="ref-healy2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">39 </div><div class="csl-right-inline">Healy J, McInnes L. <a href="https://doi.org/10.1038/s43586-024-00363-x">Uniform manifold approximation and projection</a>. <em>Nature Reviews Methods Primers</em> 2024; <strong>4</strong>: 82.</div>
</div>
<div id="ref-cohn2021" class="csl-entry" role="listitem">
<div class="csl-left-margin">40 </div><div class="csl-right-inline">Cohn R, Holm E. <a href="https://doi.org/10.1007/s40192-021-00205-8">Unsupervised machine learning via transfer learning and k-means clustering to classify materials image data</a>. <em>Integrating Materials and Manufacturing Innovation</em> 2021; <strong>10</strong>: 231–44.</div>
</div>
<div id="ref-dietrich2016" class="csl-entry" role="listitem">
<div class="csl-left-margin">41 </div><div class="csl-right-inline">Dietrich S, Floegel A, Troll M, <em>et al.</em> <a href="https://doi.org/10.1093/ije/dyw145">Random survival forest in practice: A method for modelling complex metabolomics data in time to event analysis</a>. <em>International Journal of Epidemiology</em> 2016; <strong>45</strong>: 1406–20.</div>
</div>
<div id="ref-collins2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">42 </div><div class="csl-right-inline">Collins GS, Moons KGM, Dhiman P, <em>et al.</em> <a href="https://doi.org/10.1136/bmj-2023-078378">TRIPOD+AI statement: Updated guidance for reporting clinical prediction models that use regression or machine learning methods</a>. <em>BMJ</em> 2024; <strong>385</strong>: e078378.</div>
</div>
<div id="ref-lopez-ayala2025" class="csl-entry" role="listitem">
<div class="csl-left-margin">43 </div><div class="csl-right-inline">Lopez-Ayala P, Riley RD, Collins GS, Zimmermann T. <a href="https://doi.org/10.1136/bmj-2024-082440">Dealing with continuous variables and modelling non-linear associations in healthcare data: Practical guide</a>. <em>BMJ</em> 2025; <strong>390</strong>: e082440.</div>
</div>
<div id="ref-collins2024a" class="csl-entry" role="listitem">
<div class="csl-left-margin">44 </div><div class="csl-right-inline">Collins GS, Dhiman P, Ma J, <em>et al.</em> <a href="https://doi.org/10.1136/bmj-2023-074819">Evaluation of clinical prediction models (part 1): From development to external validation</a>. <em>BMJ</em> 2024; <strong>384</strong>: e074819.</div>
</div>
<div id="ref-riley2024" class="csl-entry" role="listitem">
<div class="csl-left-margin">45 </div><div class="csl-right-inline">Riley RD, Archer L, Snell KIE, <em>et al.</em> <a href="https://doi.org/10.1136/bmj-2023-074820">Evaluation of clinical prediction models (part 2): How to undertake an external validation study</a>. <em>BMJ</em> 2024; <strong>384</strong>: e074820.</div>
</div>
<div id="ref-bycroft2018" class="csl-entry" role="listitem">
<div class="csl-left-margin">46 </div><div class="csl-right-inline">Bycroft C, Freeman C, Petkova D, <em>et al.</em> <a href="https://doi.org/10.1038/s41586-018-0579-z">The UK biobank resource with deep phenotyping and genomic data</a>. <em>Nature</em> 2018; <strong>562</strong>: 203–9.</div>
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>